### DISCUSSION TITLE
Getting Started Documentation

### DISCUSSION THREAD

**Discussion by NigelWSewell** [eclipse-langium/langium#1512] [General]
The Getting Started documentation seems to be off line. Is this Due to a website error or the docs currently getting re-written? 

https://langium.org/docs/getting-started/

![image](https://github.com/eclipse-langium/langium/assets/90779026/db4abc93-60b5-474e-9680-f6f94e19fd99)

**Comment by cdietrich** [comment]
https://github.com/eclipse-langium/langium-website/commit/41ee937ab9b42acc08054ba3c569d4ab8e1d1d8d#diff-c245378046878806169504f43569c139bc51cae57c0f9502d6d569a044a58a83

seems to have moved it but also partially vanished it

**Reply by msujew** [reply]
Nothing vanished, it was just split up. The entry point to the documentation is now available at https://langium.org/docs/introduction/. We still need to adjust our links to the new documentation structure.

The getting started guide can be found here https://langium.org/docs/learn/worflow/.

**Reply by cdietrich** [reply]
> After installing Yeoman, you can scaffold a new Langium project.

there was more there before

**Reply by msujew** [reply]
Have you clicked on the [associated link](https://langium.org/docs/learn/workflow/scaffold/) yet? :)

**Reply by cdietrich** [reply]
yes.

but there is no

`yo langium` anymore

**Reply by cdietrich** [reply]
argg i am just blind. the other header is clickable too.

**Reply by montymxb** [reply]
We should add a redirect here to the new content, or something that at least says this was moved. Beyond that the stock 404 doesn't look very good , we should consider updating that as well.

**Reply by NigelWSewell** [reply]
> We should add a redirect here to the new content, or something that at least says this was moved. Beyond that the stock 404 doesn't look very good , we should consider updating that as well.

Agreed. A broken link or 2 says 'unmaintained' to a lot of people.

**Reply by montymxb** [reply]
Yep, I've already created an issue https://github.com/eclipse-langium/langium-website/issues/236 on the website repo to focus on that.
